recipe/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+      -G "Ninja" \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=True \
+      -DCMAKE_CXX_STANDARD=17
```

Comment:
There is any specific reason why this was necessary? In theory if the project has a minimum required C++ version, this should be accounted as part of the CMake script.

Reply:
I copied this blindly from https://github.com/conda-forge/libignition-common-feedstock/blob/master/recipe/build.sh

Maybe @wolfv can comment on this

Reply:
(or @seanyen actually as he has written this build script)

Reply:
Ack, no problem then. However I think that everything should probably work even by removing that option.

Reply:
Theoretically all of conda-forge uses c++17 by default. I would also guess that this isn't strictly necessary.